### ISSUE TITLE
Override call type dialog should show current call type at first time open

### ISSUE THREAD

**Issue by op2786** [Vector35/binaryninja-api#4833]
**Version and Platform (required):**
- Binary Ninja Version: 3.6.4719-dev, dec72a07
- OS: macos
- OS Version: 14.2
- CPU Architecture: arm64

**Bug Description:**

Let's say there is a function call in linear view like this:

```
job->callback(job, config, job->field_10)
```

When I try to override call type for this call, dialog show me this as default call type: `void calltarget()`. But obviously there is different call type detected by BN. Instead of `void calltarget()`, BN should show me current call type which is something like this: `void (* callback)(struct my_job*, struct config*, int64_t)`.

If BN do so and it predicts it correctly, I can easily copy that and set structure member type to that.

**Comment by xusheng6** [comment]
Related to: https://github.com/Vector35/binaryninja-api/issues/4739

**Comment by xusheng6** [comment]
It seems to me that we are probably only failing to populate the current call type when the user uses the "set call type" for the very first time. Which I am still yet trying to reproduce. @op2786 would it be possible to provide your file/database?

According to me test, if there is already a overridden call type on the function call, we can populate it just fine.

**Comment by op2786** [comment]
Yeah you're right @xusheng6. It is only happens on the very first time. So the issue only reproduces on fist time use of the feature.

My suggestion is as seen in the example, BN shows there is a call that have 3 parameters so when I try to first time set call type for that call, it should show me the that call prototype instead of `void calltarget()`.
